thali/NextGeneration/thaliMobileNative.js

```diff
@@ -297,6 +279,118 @@
 // jscs:enable maximumLineLength
 
 /**
+ * @external "Mobile('multiConnect')"
+ * @public
+ */
+
+// jscs:disable maximumLineLength
+/**
+ * Platforms that support `multiConnect` are able to bridge from a non-TCP
+ * transport to a native TCP listener that can accept arbitrary numbers of
+ * connections. This is different than `connect` platforms whose native
+ * TCP listener can handle exactly one connection at a time and therefore need
+ * to use a multiplex layer in Node.
+ *
+ * This method MUST return an error if called while start listening for
+ * advertisements is not active.
+ *
+ * The node layer is responsible for making sure there is not more than a single
+ * outstanding `connect` or 'disconnect' method call at a time for any given
+ * peerID. If that restriction is violated then the system enters an unknown
+ * state.
+ *
+ * A call to `multiConnect` will immediately return with no information other
```

Comment:
@yaronyg Why should we go in a such long way?
I think it would be better if `Mobile('multiConnect').callNative` could directly pass the error(if any) and result to the callback.
It could be something like that
```
Mobile('multiConnect).callNative((err, data) => { 
  if (err) {
    ...
  }
  // work with data
});
```